modules/gdnative/godot/string.cpp

```diff
@@ -538,6 +538,38 @@ godot_string GDAPI godot_string_get_slicec(const godot_string *p_self, wchar_t p
 	return result;
 };
 
+godot_array GDAPI godot_string_split(const godot_string *p_self, const godot_string *p_splitter) {
```

Comment:
Hey @karroffel

Is it the right way to do a `Vector<String>` to `godot_array` conversion ? :)